src/plone/base/interfaces/controlpanel.py

```diff
@@ -1195,10 +1195,23 @@ class ISiteSchema(Interface):
         required=False,
     )
 
+    webstats_head_js = schema.SourceText(
```

Comment:
Well, since the existing is named webstat_js and I didn't want to change that fields name I came up with this fieldname to communicate their relation. If we name this one inject_head, than would you rename webstats_js to inject? That doesn't make sense to me, sorry. ;)

In an ideal world it would be refactored to names like inject_javascript and inject_javascript_head. I hope other reviewers can live with it being this way.

Reply:
fair enough